azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -7,32 +7,33 @@
 from azure.kusto.data import KustoClient
 from azure.kusto.data._models import KustoResultTable
 
-_URI_FORMAT = re.compile("https://(\\w+).(queue|blob|table).core.windows.net/([\\w,-]+)\\?(.*)")
+_URI_FORMAT = re.compile("https://(\\w+).(queue|blob|table).(core.\\w+.net)/([\\w,-]+)\\?(.*)")
```

Comment:
for all uses of \w, are we sure dashes and perhaps other characters are not allowed?

Reply:
AFAIK other characters are not allowed, but I don't know all independent clouds suffixes..